packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,176 @@ related: ['Button']
 import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 
-A **modal** loads and displays important information without navigating away from the current page.
+A **modal** displays important information to a user without requiring them to navigate to a new page.
 
 ## Primary elements
-<img src="./img/modal.png" alt="Modal" />
+
+<img src="./img/modal-call-outs.png" alt="An example of a modal in context with call outs to highlight each listed element" width="756"/>
 
 1. **Backdrop**
 2. **Modal box**
-3. **Title:** Concisely convey the purpose of the modal.
-4. **Close**
+3. **Headline**
+4. **Close icon**
 5. **Content area**
-6. **Buttons:** Enable a user to confirm, submit, or exit. Modal dialogs can include multiple action buttons, but should be limited to one primary button
+6. **Buttons** 
+
+See the [Content](#content) section for information about writing modal copy.
+
+## Spacing
+
+### Compact (24px)
+
+<img src="./img/modal-compact.png" alt="A compact modal with 24px spacers" width="487" />
+
+
+Compact modals are best used to display small amounts of content. 
+
+Their primary spacing of 24px separates:
+
+- Each edge of the modal box from all modal content.
+- The content area from both buttons.
+
+A 16px spacing separates the modal's headline from the content area. 16px also separates each button.
+
+### Large (32 px)
 
-### Spacing
-#### Compact
-<img src="./img/compact.png" alt="Compact" /> 
+<img src="./img/modal-large.png" alt="A large modal with 32px spacers" width="605" />
 
-The compact modal dialog uses a spacing of 24px. Use this for smaller modals that don’t contain much content.
+Large modals accommodate a larger content area for elements like code previews, forms, and tables.
 
-#### Large
-<img src="./img/large.png" alt="Large" />
+Their primary spacing of 32px separates:
 
-The large modal dialog uses a spacing of 32px. Use this for larger modals that contain a lot of content (code previews, forms, tables...).
+- Each edge of the modal box from all modal content.
+- The content area from both buttons.
+
+A 24px spacing separates the modal's headline from the content area, and a 16px spacing separates each button.
+
+## Placement
+
+A modal can be placed in two positions on a page:
+1. [Center-aligned (default)](/components/modal/html-demos/basic/): The modal appears horizontally and vertically aligned at the center of a page.
+2. [Top-aligned](/components/modal/html-demos/top-aligned/): The modal appears horizontally and vertically aligned at the top of a page.
+
+### Center-aligned modal (default)
+
+<img src="./img/center-aligned-modal.png" alt="A modal pictured in center alignment on a page" />
+
+Default to centering a modal horizontally and vertically on the page.
```

Comment:
Could you move this text to be above the image, i'm worried people will miss it underneath the image